source/retryable-writes/retryable-writes.rst

```diff
@@ -66,7 +66,23 @@ ClientSession
    name of this object MAY vary across drivers.
 
 Retryable Error
-   An error is considered retryable if it meets any of the following criteria:
+   An error is considered retryable if it has a RetryableWriteError label in
+   its top-level "errorLabels" field.
+
+   For server versions 4.3 and newer, MongoDB will add a RetryableWriteError label to
+   errors or server responses that it considers retryable before returning them to the
+   driver. As new server versions are released, the errors that are labeled with the
+   RetryableWriteError may change.
+
+   There are some cases where the driver MUST add the RetryableWriteError label to
+   an error or server response from a supported retryable write operation:
+   - any network exception
+   - any error with an error message containing the substrings "not master"
```

Comment:
Did we clarify whether this was still needed for 3.6+ servers?

Reply:
I asked whether we still needed to handle this case in the replication channel and got this response:
> siyuan.zhou 2:45 PM
> I’m afraid you’d have to audit the code to answer that.

Judah and Lingzhi don't know either. Should I make a server ticket to try and get this done? Should I try to figure it out myself? Is it worth spending time on right now?

Reply:
We can keep this as-is in this PR. A new spec ticket to request investigation seems fine, too, even if we ultimately never budget time for it.

Reply:
Here's the SPEC ticket: [SPEC-1477](https://jira.mongodb.org/browse/SPEC-1477). I've linked it to SPEC-1420 and DRIVERS-525 for visibility.